src/data/openings.json

```diff
@@ -36,6 +36,32 @@
     ],
     "applicationUrl": "https://docs.google.com/forms/d/e/1FAIpQLSf4Mk-vxO3Nl193LkCSGPOrrT5uDvA27peH8du7hSnfUsxb0A/viewform"
   },
+  {
+    "openingId": "senior-seo-specialist",
+    "category": "Engineering",
```

Comment:
Please change the category to **Marketing** for this role

Reply:
Yes, I have updated it.